public/mycoportal-collections.csv

```diff
@@ -0,0 +1,167 @@
+collid,coll acryonym,coll name
```

Comment:
@mo-nathan 
1. You suggested moving the MCP_COLLECTIONS out of herbaria.rb and into the db.
This PR removes it from herbaria.rb, and replaces it this file received from Andy Miller.
I think that periodically updating that file would be a relatively easy way to keep things up to date.
Is there a better approach?
2. if a periodically update file is the right approach, where should the file live?